README.md

```diff
@@ -56,6 +56,37 @@ Re.Pack uses Webpack 5 and React Native CLI's plugin system to allow you to bund
 - [x] Code signing and verification
 - [x] `@callstack/repack-init` - tool that automates the integration of the Re.Pack into React-Native projects
 
+### Planned features
+
+Available at: [Projects](https://github.com/callstack/repack/projects?type=classic)
+
+- [ ] `ChunksToHermesBytecodePlugin` plugin to automatically transform async chunks to bytecode format.
+- [ ] Code signing and verification
+- [ ] `webpack-init` command
```

Comment:
This whole section is irrelevant – these features are already implemented & released. Either this branch needs sync with `main,` or conflicts were resolved in a wrong way